src/layout-grid/cell.js

```diff
@@ -28,13 +28,14 @@ export default function Cell({
     overrides.Cell,
     DefaultStyledCell,
   );
+  const gridContext = React.useContext(GridContext);
   return (
     <StyledCell
       $align={align}
-      $gridColumns={gridColumns}
-      $gridGaps={gridGaps}
-      $gridGutters={gridGutters}
-      $gridUnit={gridUnit}
+      $gridColumns={gridColumns || gridContext.gridColumns}
+      $gridGaps={gridGaps || gridContext.gridGaps}
+      $gridGutters={gridGutters || gridContext.gridGutters}
+      $gridUnit={gridUnit || gridContext.gridUnit}
```

Comment:
Do we need the `||` here? There isn't any reason to pass values that differ from the grid context. In that case, we can remove these `gridXYZ` props from `Cell` altogether.

Reply:
I did this so that it would be entirely backwards compatible with the current Cell component. I don't really see why someone would want to pass the grid props directly into a Cell component, but if we remove the props it turns this PR into a breaking change. Am happy to do it if you think it's worth making a breaking change?

Reply:
Good point, can you just add a comment here to remind us to clean it up in v10? Something like this:
```
// TODO(v10): Remove gridXYZ props
```

Reply:
OK, added in the latest commit
https://github.com/uber/baseweb/pull/3264/commits/d7e77300056b7da07ca428911848ed508850102d